forge-gui/res/cardsfolder/upcoming/kaylas_music_box.txt

```diff
@@ -1,9 +1,10 @@
 Name:Kayla's Music Box
 ManaCost:2
 Types:Legendary Artifact
-A:AB$ Dig | Cost$ W T | DigNum$ 1 | ChangeNum$ All | DestinationZone$ Exile | RememberChanged$ True | ExileFaceDown$ True | SpellDescription$ Look at the top card of your library, then exile it face down. (You may look at it any time.) 
-A:AB$ Effect | Cost$ T | RememberObjects$ Remembered | StaticAbilities$ MayPlay | SpellDescription$ Until end of turn, you may play cards you own exiled with CARDNAME.
-SVar:MayPlay:Mode$ Continuous | EffectZone$ Command | Affected$ Card.IsRemembered | AffectedZone$ Exile | MayPlay$ True | Description$ Until end of turn, you may play cards you own exiled with CARDNAME.
+A:AB$ Dig | Cost$ W T | DigNum$ 1 | ForceRevealToController$ True | ChangeNum$ All | DestinationZone$ Exile | ExileFaceDown$ True | SpellDescription$ Look at the top card of your library, then exile it face down. (You may look at it any time.) 
+S:Mode$ Continuous | MayLookAt$ You | Affected$ Card.ExiledWithEffectSource | AffectedZone$ Exile
```

Comment:
Your're not calling this from an effect, so it should only be `ExiledWithSource`

Reply:
Good point, I guess line 8 and 9 can go since there is no more rememberedobject  ?